frontend/app/services/activity-service.js

```diff
@@ -51,6 +55,16 @@ module.exports = function(HALAPIResource, $http, PathHelper){
       };
 
       return activity.links.update.fetch(options).then(function(activity){
+        notification = NotificationsService.addSuccess(
+          I18n.t('js.work_packages.comment_updated')
+        );
+        // Remove the notification some time later
+        // but only when we're not in accessible mode
+        if (!ConfigurationService.accessibilityModeEnabled()) {
+          $timeout(function() {
+            NotificationsService.remove(notification);
+          }, 5000);
```

Comment:
I think this is the wrong place to handle the disappearing behaviour of notifications.

This should be handled globally by the NotificationService/NotificationDirective to create a behaviour that is consistent. As such, I think it is outside the scope of this PR.